### ISSUE TITLE
Suggestions for new board enclosure

### ISSUE THREAD

**Issue by Dovydas-R** [luftek/LimeSDR-USB#15]
Great job but I have some suggestions:
1. Add additional SMA hole for REF_CLK_OUT
2. Rename existing REF_CLK to REF_CLK_IN
3. Add DC connector polarity information
4. Current case version hides board LEDs.
   It is possible to connect external LEDs to LimeSDR-USB board so you can put LEDs on case panel and connect them using pinheader connector to LimeSDR-USB board.
5. Possible thermal issues due to poor thermal conductivity of board edges and aluminum enclosure.
6. To improve thermal conditions add ventilation holes and prepare mounting for fan.

**Comment by zigastegu** [comment]
1. agree
2. agree
3. agree, was just thinking about that, usually it is positive polarity, but not always the case (Gameboy DMG has negative)
4. #12  I am not sure to place them on rear panel, I would rather place them on front
5. and 6. are currently out of my reach (don't have LimeSDR yet, waiting on campaign delivery) some holes are added on rear panel #7

**Comment by zigastegu** [comment]
So, I have made changes for first 4 suggestions:
1., 2. done, 2nd CLK io added
3. added symbol for positive polarity, Walter Lain's lib [http://smisioto.no-ip.org/elettronica/kicad/kicad-en.htm](http://smisioto.no-ip.org/elettronica/kicad/kicad-en.htm)
4. added 3d models from Dialight, BUT still not sure if LED holes should be predrilled.

![2016-10-30 18_29_40-freecad](https://cloud.githubusercontent.com/assets/1379063/19838449/b2c73f4a-9ecf-11e6-8145-b9220ff07e77.png)
![2016-10-30 18_30_14-program manager](https://cloud.githubusercontent.com/assets/1379063/19838448/b2c6a0ee-9ecf-11e6-8909-cd6161d7c4de.png)
![2016-10-30 18_30_48-3d viewer](https://cloud.githubusercontent.com/assets/1379063/19838447/b2c678da-9ecf-11e6-927a-877a8ad4c18f.png)

**Comment by zigastegu** [comment]
https://github.com/myriadrf/LimeSDR-USB/issues/8

**Comment by zigastegu** [comment]
![image](https://cloud.githubusercontent.com/assets/1379063/20017204/e7569016-a2c2-11e6-8fe8-460122e9ac52.png)
![image](https://cloud.githubusercontent.com/assets/1379063/20017227/f66fff4c-a2c2-11e6-81cb-8bf011c93cac.png)

**Comment by Dovydas-R** [comment]
Looks OK

**Comment by zigastegu** [comment]
![image](https://cloud.githubusercontent.com/assets/1379063/20069558/621e5af2-a51d-11e6-8923-3fb9d445ec91.png)

**Comment by Dovydas-R** [comment]
Several weeks ago I received enclosure from Ziga (Luftek). Thanks!

Precisely made front and back PCB panels looks great.

There was two different panels for RF: 
- minimal with 4 holes for SMA connectors (one band for each channel TX, RX) 
- full with with 10 holes for SMA connectors

LimeSDR board fits perfectly.

Some thermal tests we performed:
- Board withot enclosure and without fan reached 71 °C temperature after 1 hour of operating.
- Same board placed in Luftet case reched 76 °C after 1 hour of operating.

So thermal condistions for board in not perfect.
I also really missed any LED indicators on panel.
If all earlier mentioned sugestions will be implemented, this enclosure could be good alternative for other LimeSDR enclosures.

I'm attaching some pictures.

![1](https://cloud.githubusercontent.com/assets/19203153/20643110/aed54ee0-b428-11e6-803f-2596877b4f96.JPG)
![2](https://cloud.githubusercontent.com/assets/19203153/20643112/b3ccf222-b428-11e6-9393-94ade6ad7a2a.JPG)
![3](https://cloud.githubusercontent.com/assets/19203153/20643113/b5f8a0a0-b428-11e6-83de-a274d472b581.JPG)
![4](https://cloud.githubusercontent.com/assets/19203153/20643114/b7817898-b428-11e6-9851-2d5c0828bffd.JPG)
![5](https://cloud.githubusercontent.com/assets/19203153/20643115/b8e83938-b428-11e6-9570-5a46e6349073.JPG)
![6](https://cloud.githubusercontent.com/assets/19203153/20643116/ba69b822-b428-11e6-9d5e-ac539d6787dc.JPG)

**Comment by vince48** [comment]
thanks for sharing

Vincent

**Comment by zigastegu** [comment]
Thank you on photos. Sry did not include cables.

- How did this test look like? Test points, distriburion of heat?
- Any conclusion wether we need to siphone this heat away? What are temp spec. of RF chip, thermal shutdown at?

Optios:
1. Scrape soldermask from front panel and solder LimeSDR to front panel for bigger heat disipation surface.
2. Add aluminum pillars between chips and top shell
3. Heatsink + fan + additional holes

Maybe we can write answers here, in comments below.

**Comment by Dovydas-R** [comment]
Board RF RX1 channel was receiving on 80 MSPs for 1 hour. Temperature was measured using onboard temperature sensor. This is not maximum board load.

In my opinion the normal board board operating temperature  is up to 60°C.

I think best way to reduce temperature inside would be use of FAN, mounted on the top. This needs to make holes in the case. It would be very nice to have fan guard (grill). Holes and guard can be made by one machining operation in aluminum case. Board has connector for FAN (3,3V) and is temperature controlled. Of course, the enclosure must have additional holes to ensure proper air circulation (inlet and outlet, outside -> inside -> outside). Fan  example: http://www.digikey.com/product-detail/en/sunon-fans/MC30060V2-000U-A99/259-1572-ND/2757810

![s-l300](https://cloud.githubusercontent.com/assets/19203153/20664236/fda6b44c-b562-11e6-8643-f3c20e43b674.jpg)

Another not so effective solution could be heat transfer from board edges to aluminum case. This needs good thermal contact between board and case.

**Comment by zigastegu** [comment]
QFN 11.5×11.5 mm package
typical 880mW in full 2x2 MIMO mode (550mW in SISO mode) using external LDOs
Operating Temperature	-40°C ~ 85°C

Yes, we need something to cool it, 1W is some power, probably other chips also contribute to heat generation.

2. option
@Dovydas-R  do you have an option to purchase aluminium rod and some fan paste?
I checked and length between chip and top shell of enclosure is approx. 26 mm (25.4 mm / 1 inch) else I will prepare and send it to you for testing
![image](https://cloud.githubusercontent.com/assets/1379063/20773500/67c44f88-b752-11e6-89ff-9f5e35802991.png)

**Comment by Dovydas-R** [comment]
Not only transceiver chip (LMS7002) produces heat when operating, but also FPGA, RAMs, LDOs etc.
Rod between enclosure and one or more chips will be complicated and not so effective solution.

FAN on the top of the board was tested and it is simple and effective solution.

**Comment by zigastegu** [comment]
Can you share test setup where you used fan on top of the board?
Was board "free" or in any enclosure?

I will check for aluminum/brass heatsinks, maybe compatible same as RPi

**Comment by luftek** [comment]
https://www.ettus.com/content/images/B205mini-i_heat_sink_2_2.jpg

**Comment by luftek** [comment]
Thermal Camera in Customs, hopefully arrives next week.
Then need to wait for LimeSDR to arrive for actual thermal testing 👍

**Comment by luftek** [comment]
http://www.ebay.com/itm/-/122582037877?

**Comment by zigastegu** [comment]
Fan, heatsinks, heatpad used in version 1v0
Commit: dc9700f
Kit sold on eBay: http://www.ebay.com/itm/-/122582037877
